monkey

```diff
@@ -8,7 +8,7 @@
   <h1>Oooh ooh ah ah monkeying around</h1>
   <hr>
   
-  <p style="background-color:lawngreen"> funky monkey</p>
```

Comment:
have you considered changing this to `chunky monkey` instead?

Reply:
good idea